config.py

```diff
@@ -19,7 +20,6 @@ class ProductionConfig(Config):
 
 
 class StagingConfig(Config):
-    DEVELOPMENT = True
```

Comment:
I don't think staging environment needs to skip authorization... Only for development and tests

Reply:
that's the reason why I removed the Development=true from the staging..